src/translation/german.c

```diff
@@ -587,7 +602,11 @@ static translation_string all_strings[] = {
     {TR_FIGURE_TYPE_TOURIST,  "Reisender"},
     {TR_SAVEGAME_LARGER_VERSION_TITLE, "Inkompatible Speicherstand-Version"},
     {TR_SAVEGAME_LARGER_VERSION_MESSAGE, "Der Speicherstand, den Sie zu laden versuchen, ist für eine neuere Version von Augustus.\n Bitte aktualisieren Sie Augustus auf die neueste Version."},
-    {TR_BUILDING_COLONNADE, "Kolonnade"},
+    {TR_SAVEGAME_NOT_ABLE_TO_SAVE_TITLE, "Spiel kann nicht gespeichert werden" },
+    {TR_SAVEGAME_NOT_ABLE_TO_SAVE_MESSAGE, "Es war nicht möglich, das Spiel zu speichern.\n
```

Comment:
When you have multiline strings, please end the old line with `"` and start the new one also with `"`, otherwise some compilers won't like it.